backend/src/api/volunteer/content-types/volunteer/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "kind": "collectionType",
+  "collectionName": "volunteers",
+  "info": {
+    "singularName": "volunteer",
+    "pluralName": "volunteers",
+    "displayName": "Volunteer",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "Content": {
+      "type": "richtext"
+    },
+    "ApplicationLink": {
```

Comment:
Really nice idea to separate this from the main content, as yes because it's a google form link we might want to change it. Also nice that we can integrate it with our own styled button in the future